src/launch/lib/automation/provider/az/functions.py

```diff
@@ -48,3 +45,5 @@ def deploy_remote_state(
         subprocess.run(run_list, check=True)
```

Comment:
For what it's worth, we can probably cut down on some of the path changing with the use of the `cwd` kwarg on subprocess.run. Don't need to change anything for this PR, but going forward I think that is a much cleaner solution that doesn't result in unexpected directory changes if there's an unhandled exception somewhere other than in the subprocess call.